lib/src/install.rs

```diff
@@ -1245,11 +1245,8 @@ pub(crate) async fn install_to_disk(mut opts: InstallToDiskOpts) -> Result<()> {
     // At this point, all other threads should be gone.
     if let Some(state) = Arc::into_inner(state) {
         // If we had invoked `setenforce 0`, then let's re-enable it.
-        match state.selinux_state {
-            SELinuxFinalState::Enabled(Some(guard)) => {
-                guard.consume()?;
-            }
-            _ => {}
+        if let SELinuxFinalState::Enabled(Some(guard)) = state.selinux_state {
```

Comment:
While we're here...I think a further improvement is to make this a method on `State`, i.e. `impl State { fn consume(self) -> Result<()> }` which just calls this code (to better encapsulate things).